awslabs/performance-dashboard-on-aws
# GTT-721 Re-styling create and edit dashboard pages

## Description

Applying the new style to the Create Dashboard and Edit Dashboard pages. 

## Testing

You can test in my personal environment both screens: 
- Create dashboard: https://fdingler.badger.wwps.aws.dev/admin/dashboard/create
- Edit dashboard: https://fdingler.badger.wwps.aws.dev/admin/dashboard/edit/3f151f01-ce83-43ba-a000-43278f9d3c3e/details

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
